python-client/giskard/scanner/robustness/unit_perturbation.py

```diff
@@ -0,0 +1,170 @@
+import pandas as pd
+from giskard.ml_worker.testing.registry.transformation_function import transformation_function
+import random
+
+
+class TransformationGenerator:
+    def __init__(self, model, dataset):
+        self.model = model
+        self.dataset = dataset
+        self.column_types = dataset.column_types
+        self.text_perturbation_generator = TextTransformer()
+
+    def generate_std_transformation(self, feature):
+        mad = self.dataset.df[feature].mad()
```

Comment:
pandas `mad` is deprecated and it is not what we want. We want "median absolute deviation" which is a robust estimator of the standard deviation, and not the "mean absolute deviation".

σ ≈ 1.4826 * MAD

where MAD is `median(abs(x - median(x)))`.